src/PaymentGateways/Gateways/TestGateway/testGateway.tsx

```diff
@@ -0,0 +1,50 @@
+import {__} from '@wordpress/i18n';
+//import type {FormData, Gateway} from '@givewp/forms/types';
+
+//TODO: update with types when they are available
+const gateway = {
+    id: 'test-gateway',
+    async beforeCreatePayment(values) {
+        if (values.firstName === 'error') {
+            throw new Error('Failed in some way');
+        }
```

Comment:
Oh, this is a nice little tool. Let's make sure this is documented in some way, maybe in the TestGateway class docblock.

Reply:
Yeah it's pretty nifty for testing error messages.  I've been thinking about doing more stuff like this to the test gateway to see different responses. I just added a comment here for now 41d7be7a